CommonData/ipcchannel.cpp

```diff
@@ -251,27 +250,21 @@ bool IPCChannel::jaspAlive()
 {
 	if(!_isSlave)
 		return true; //if jasp asks its obviously alive
-
-	ifstream in;
-	in.open(_jaspHeartBeatPath);
-	if(!in.is_open()) {
-		Log::log() << "Could not find heartbeat file" << std::endl;
-		return false;
-	}
-
-	uint64_t stamp;
-	in >> stamp;
-	in.close();
-
+	
+	int64_t newTimestamp = Utils::getFileModificationTime(_jaspHeartBeatPath);
```

Comment:
```suggestion
	int64_t stamp = Utils::getFileModificationTime(_jaspHeartBeatPath);
```

This should just be stamp? or bellow should be changed, cause `stamp` was not defined: 
```
if(stamp != _lastHeartBeatTimestamp) {
		_lastHeartBeatTimestamp = stamp;
		return true;
	}
```